src/options.tsx

```diff
@@ -60,13 +70,23 @@ function BasicSettings() {
         </label>
 
         <input
-          className="bg-gray-50 border w-64 border-gray-300 text-gray-900 text-sm rounded-lg 
+          className="bg-gray-50 border w-64 border-gray-300 text-gray-900 text-sm rounded-lg
               focus:ring-blue-500 focus:border-blue-500 block"
           value={apiURL}
           onChange={updateApiURL}
           id="api_url"
         />
       </div>
+      <div className="flex flex-col gap-y-2">
```

Comment:
Appreciate it if there can be an empty line break two parts of code

Reply:
thanks! done😁